lld/wasm/Driver.cpp

```diff
@@ -47,6 +47,16 @@ namespace lld::wasm {
 Configuration *config;
 Ctx ctx;
 
+void Ctx::reset() {
+  objectFiles.clear();
```

Comment:
Nit: shouldn't we reset *all* the fields, including the `bool` fields? (And maybe initialize `isPic` to some explicit value rather than leaving it as uninited?)